styles/styles.css

```diff
@@ -293,7 +293,7 @@ main {
 }
 
 main .section {
-  padding-top: 0 30px 65px 30px;
+  padding: 0 30px 65px;
```

Comment:
this was needed to fix linting. this was checked in to main without lint check @cuzuco2